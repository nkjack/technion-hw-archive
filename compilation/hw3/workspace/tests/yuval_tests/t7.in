void main(){
  int x;
}

void checkInt(){
  int x;
  byte z = 6 b;
  int y = 19;
  y = z;
  x = true;
}