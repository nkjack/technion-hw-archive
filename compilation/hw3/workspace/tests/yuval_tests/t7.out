---end scope---
main - 0 preconditions
x INT 0
line 10: type mismatch
